docs/release_notes.rst

```diff
@@ -20,7 +20,7 @@ Bug Fixes
   The bug caused some argument names to go unreported in D417 (#448).
 * Fixed an issue where skipping errors on module level docstring via #noqa
   failed when there where more prior comments (#446).
-
+* Correctly detect publicity of modules inside directories (#470, #493).
```

Comment:
the PR number seems incorrect?

Reply:
Woops, I had thought these were GitHub issue numbers.

Let me update this `#494`.